content/blog/2018-06-26-roomba.Rmd

```diff
@@ -0,0 +1,107 @@
+---
+slug: "roomba"
+title: A package for tidying nested lists
+package_version: 0.0.1
+authors:
+  - name: Amanda Dobbyn
+    url: https://github.com/aedobbyn
+  - name: Jim Hester
+    url: https://github.com/jimhester
+  - name: Laura DeCicco
+    url: https://github.com/ldecicco-USGS
+  - name: Christine Stawitz
+    url: https://github.com/cstawitz
+  - name: Isabella Velasquez 
+    url: https://github.com/ivelasq
+date: 2018-06-26
+categories: blog
+topicid: 925
+tags:
+- r
+- community
+- software
+- package
+- meetings
+- unconf
+- unconf18
+---
+Data == knowledge! Much of the data we use, whether it be from government repositories, social media, GitHub, or e-commerce sites comes from public-facing APIs. The quantity of data available is truly staggering, but munging JSON output into a format that is easily analyzable in R is an equally staggering undertaking. When JSON is turned into an R object, it usually becomes a deeply nested list riddled with missing values that is difficult to untangle into a tidy format. Moreover, every API presents its own challenges; code you've written to clean up data from GitHub isn't necessarily going to work on Twitter data, as each API spews data out in its own unique, headache-inducing nested list structure. To ease and generalize this process, [Amanda Dobbyn proposed](https://github.com/ropensci/unconf18/issues/24) an #unconf18 project for a general API response tidier! Welcome `roomba`, our first stab at easing the process of tidying nested lists!

```

Comment:
In place of "...an #unconf18 project..." best text and link for unconf in this para is probably
`[unconf18](https://ropensci.org/blog/2018/06/05/unconf18/`)

because that post will link to all other posts about unconf

Reply:
done!